cfme/tests/services/test_service_catalogs.py

```diff
@@ -202,3 +204,35 @@ def test_advanced_search_registry_element(request, appliance):
     request.addfinalizer(view.search.close_advanced_search)
     view.search.advanced_search_form.search_exp_editor.registry_form_view.fill({'type': "Registry"})
     assert view.search.advanced_search_form.search_exp_editor.registry_form_view.type.is_displayed
+
+
+@pytest.mark.tier(2)
+@pytest.mark.provider([RHEVMProvider], selector=ONE)
+def test_order_service_after_deleting_provider(appliance, provider, setup_provider, catalog_item):
+    """
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.8
+        tags: service
+    """
+    template_name = catalog_item.prov_data["catalog"]["catalog_name"]["name"]
+    template_id = appliance.rest_api.collections.templates.find_by(name=template_name)[0].id
```

Comment:
If the template does not exist on the provider can we skip the test?

Reply:
@john-dupuy I'm retrieving this from fixture return l.e. `catalog_item` for finding `id` .
`catalog_item` fixture using `provisioning`; which going to call first.

https://github.com/ManageIQ/integration_tests/blob/1f5035262ae7d4c5f43e6fe1f33276ce5b7b4236/cfme/fixtures/service_fixtures.py#L50

It should taken care by `provisioning`; which checking for `provisioning` key in `yaml`